js/src/Ice/InstanceExtensions.js

```diff
@@ -424,3 +438,11 @@ Instance.prototype.destroy = function () {
         });
     return promise;
 };
+
+Object.defineProperty(Instance.prototype, "clientConnectionOptions", {
+    get: function () {
+        return this._clientConnectionOptions;
+    },
+    enumerable: true,
+    configurable: true,
```

Comment:
I'm not a JS expert, but do we need `configurable` here?

Reply:
We can remove it, as we are not going to change it later.

Reply:
Fixed.